src/verb/builtin.rs

```diff
@@ -33,15 +33,17 @@ pub fn builtin_verbs() -> Vec<Verb> {
             "/bin/cp -r {file} {newpath:path-from-parent}",
             StayInBroot,
         )
-        .unwrap()
```

Comment:
My default keybindings are Tab, but I replace them with spaces, I intended to but I forgot